src/cljs/webchange/editor_v2/dialog/dialog_text_form/side_menu/main/voice_over/state.cljs

```diff
@@ -102,3 +103,8 @@
   ::bring-to-top
   (fn [{:keys [_]} [_ url]]
     {:dispatch [::state-scene/update-asset-date url (.now js/Date)]}))
+
+(re-frame/reg-event-fx
+ ::open-voice-over-audio-window
+ (fn [{:keys [db]} [_]]
+     {:dispatch-n [[::chunks/open]]}))
```

Comment:
`{:dispatch [::chunks/open]}` for single event

Reply:
Done